app/Http/Controllers/Admin/LoginController.php

```diff
@@ -82,7 +83,15 @@ public function login(LoginRequest $request)
                 $user = User::updateOrCreate($userCondition, $user);
                 # Set login for user
                 Auth::login($user, $request->filled('remember'));
-                return redirect("/admin");
+                # check redirect to
+                $checkURL = session('url_previous')[0];
```

Comment:
- this code has exception when session('url_previous') return null. please check it.